components/performance/editor/PerformanceEditor.vue

```diff
@@ -740,6 +855,54 @@ export default {
       };
 
       this.deletedDiscounts.push(discount);
+    },
+    selectDefaultRelaxedCategories() {
+      // eslint-disable-next-line vue/no-mutating-props
+      this.performance.relaxedName = 'Relaxed';
+      // Set local relaxed categories and emit update instead of mutating the prop
+      const selected = this.allRelaxedCategories.filter(
+        (rc) => rc.defaultRelaxed
+      );
+      this.setRelaxedCategories(selected);
+    },
+    selectDefaultSensoryFriendlyRelaxedCategories() {
+      // eslint-disable-next-line vue/no-mutating-props
+      this.performance.relaxedName = 'Sensory Friendly';
```

Comment:
Weirdly, I think since the props update, the name no longer shows in the "Relaxed Performance Name" box when doing an auto-fill until _after_ a save.... See the video below to see what I mean:

https://github.com/user-attachments/assets/af438757-6c70-4ff5-82ea-bf621864d5f7

Reply:
Weirdly, the "Concession Type" bit of the page seems to be broken in exactly the same way - again, that it adds concession types without showing them until the "Save" button is clicked and the page reloaded... See the video below on that (and if possible, can you investigate and fix both?). No worries though if it turns out to be a wider issue!

https://github.com/user-attachments/assets/8653acd6-278a-4b94-b154-9da278804aca